internal/cluster/cluster.go

```diff
@@ -1023,7 +1023,7 @@ func (m *Manager) createClusterDataFiles(ctx context.Context, c *common.Cluster,
 	// we don't want to stop on error
 	_ = m.uploadDataAsFile(ctx, log, cluster, fileName, objectHandler)
 
-	events, err := m.eventsHandler.V2GetEvents(ctx, c.ID, nil, nil)
+	events, _, err := m.eventsHandler.V2GetEvents(ctx, c.ID, nil, nil, common.UnlimitedEvents, common.NoOffsetEvents)
```

Comment:
What do you think about definining option object and pass that instead? That would keep signature smaller and clearer even if we add additional options. Something like:
```
type QueryOptions struct {
   PageNumber int64
   ItemsPerPage int64
}
```
maybe with better naming :sweat_smile: 
Also it could be recycled for other APIs (like paginating clusters)